rs/state_machine_tests/src/lib.rs

```diff
@@ -112,6 +111,7 @@ use ic_test_utilities_registry::{
     add_single_subnet_record, add_subnet_key_record, add_subnet_list_record, SubnetRecordBuilder,
 };
 use ic_test_utilities_time::FastForwardTimeSource;
+pub use ic_types::ingress::WasmResult;
```

Comment:
Just want to make sure that this re-export (as well as `ic_error_types::{ErrorCode, UserError}` above) are intentional.

Reply:
Yes, those are intentional as PocketIC defines those types, too.